Fix:
error: implicitly declaring library function 'isalnum' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'yylex' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'malloc' with type 'void *(unsigned long)' [-Werror,-Wimplicit-function-declaration]
--- ctlwords.c.orig	1999-02-14 17:02:25.000000000 -0600
+++ ctlwords.c	2022-01-14 01:07:32.000000000 -0600
@@ -22,7 +22,9 @@
 
 #endif
 
+#include <ctype.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 
 main(int argc, char *argv[])
--- grammar.y.orig	2008-01-18 11:43:20.000000000 -0600
+++ grammar.y	2022-01-17 03:59:10.000000000 -0600
@@ -75,6 +75,8 @@
 char *yyfilename;
 int yylineno;
 
+int yylex(void);
+
 #ifdef HAVE_STDARG_H
 /* GCC complains if we declare this function in traditional style */
 void
--- image/gif.c.orig	2022-01-17 03:55:37.000000000 -0600
+++ image/gif.c	2022-01-17 04:01:48.000000000 -0600
@@ -17,6 +17,7 @@
 
 #include "image.h"
 #include <gif_lib.h>
+#include <stdlib.h>
 
 static void
 tellAboutImage(name, gifp)
